### PR TITLE
Checkpoint 21

### PR DESCRIPTION
**Pull request by impactmass**
Styled projects section & Education section
Found places to use lesson 21 content

Preview https://s5kc6.csb.app/
CodeSandbox: https://codesandbox.io/s/focused-fog-s5kc6

@angelsvirkov take a look.

### CODE REVIEW

File: checkpoint/index.html [hunk 1]
```diff
@@ -34,38 +34,83 @@ <h2>
...
+
+  <!-- Here is the Projects section -->
+  <section class="section projects">
+    <h2>Projects</h2>
+    <ul>
+      <li class="blog-project">
+        <div class="details">
+          <h3 class="title">The Food Blog</h3>
+          <p class="description">
+            I built this website for my friend who loves cook and try out
```
Comment (angelsvirkov) [inline checkpoint/index.html:56]:
```suggestion
            I built this website for my friend who loves cooking and trying out
```

**Review by angelsvirkov — CHANGES_REQUESTED** [review]
That is some massive improvement. Thank you very much for taking care of this. I hope our students would follow it. Just one suggestion from my side

File: checkpoint/index.html (hunk 1 above)
Comment (impactmass) [reply checkpoint/index.html:56]: Good catch

**Review by angelsvirkov — APPROVED** [review]

**Comment by impactmass** [comment]
Cool. Thanks. Closing the PR, and leaving the branch up.

**Pull request closed without merging**